lib/search-bar/index.js

```diff
@@ -165,20 +181,21 @@ export default class SearchBarContainer extends Component {
         <div className='inputContainer'>
           <img src={searchIcon} className='searchIcon' alt='search'/>
           {inAutoCompleteSearch && <div className='autosuggestSpinner'><LoadingSpinner /></div>}
-          <Autosuggest
-            suggestions={suggestions}
-            onSuggestionsUpdateRequested={this.onSuggestionsUpdateRequested}
-            getSuggestionValue={getSuggestionValue}
-            renderSuggestion={renderSuggestion}
-            inputProps={inputProps}
-            focusInputOnSuggestionClick={false}
-            onSuggestionSelected={(event, { suggestion }) => this.handleOnOptionSelected(suggestion) }
-            multiSection={multiSection}
-            renderSectionTitle={renderSectionTile}
-            getSectionSuggestions={getSectionSuggestions}
-            shouldRenderSuggestions={() => true}
-            focusFirstSuggestion
-          />
+          <div onBlur={() => this.onBlur()}>
```

Comment:
Nice trick!
Guess so, but...  did you try to add the `onBlur` option tu autosuggest options? (asking just to know if it would work, I don't really know if all options are given as they are to the input)